src/fiatExchanges/FiatExchangeAmount.tsx

```diff
@@ -289,9 +254,7 @@ function FiatExchangeAmount({ route }: Props) {
       >
         <View style={styles.amountInputContainer}>
           <View>
-            <Text style={styles.exchangeBodyText}>{`${t('amount')} ${
-              currency === Currency.Celo ? '(CELO)' : ''
```

Comment:
I think this change was missed in this commit https://github.com/valora-inc/wallet/commit/7866d008145b6f35b1b47060b1c616effb6ec450

That same text is now in the header instead of here.